sys/amd64/conf/MINIMAL-NODEBUG

```diff
@@ -0,0 +1,10 @@
+#
+# MINIMAL-NODEBUG -- Non-debug MINIMAL kernel.
+#
+# This is the MINIMAL equivalent to GENERIC-NODEBUG.
```

Comment:
I think this should be annotated with

```
#NO_UNIVERSE
```

there's nothing we'll get out of compiling this during UNIVERSE